cli/azd/pkg/project/framework_service_docker.go

```diff
@@ -49,5 +57,38 @@ func NewDockerProject(config *ServiceConfig, env *environment.Environment, docke
 		env:       env,
 		docker:    docker,
 		framework: framework,
+		options:   createDockerOptions(config),
```

Comment:
Do we need to add this field?  
We are saving all the `config` in the config field already.  
We could call `createDockerOptions(config)` from the `package` method right?  

I see the benefit of parsing `config` just once to get docker options, but I can't stop looking at it as an extra source of thruth which might lead to issues in the future (if we ever need to update/mutate the options, or use them to clone the project, etc etc)

Reply:
I agree with you and made some adjustments.